components/dashboard/src/projects/NewProject.tsx

```diff
@@ -250,16 +251,18 @@ export default function NewProject() {
             <div className="h-3 border rounded-b-xl border-gray-100 bg-gray-100"></div>
         </div>);
 
-        const renderEmptyState = () => (<div className="mt-8 border rounded-xl border-gray-100 flex-col">
-            <div className="p-10">
-                <div className="p-6 text-center text-gray-500 bg-gray-100 rounded-xl">
-                    <h3 className="mb-6">
-                        Continue on GitHub
+        const renderEmptyState = () => (<div className="mt-8 border rounded-xl border-gray-100 dark:border-gray-700 flex-col">
+            <div>
+                <div className="px-12 py-16 text-center text-gray-500 bg-gray-50 dark:bg-gray-800 rounded-xl">
+                    <img src={NoAccess} title="No Access" className="m-auto mb-4" />
+                    <h3 className="mb-2 text-gray-600 dark:text-gray-400">
+                        No Access
                     </h3>
-                    <span className="">
-                        Install the GitHub app on selected Repositories to continue.
+                    <span className="dark:text-gray-500">
+                        Authorize GitHub (github.com) or select a different account.
```

Comment:
**issue**: Prompting for selecting a different account here would make more sense once we add the provider dropdown selection above this area. See specs in https://github.com/gitpod-io/gitpod/issues/4948.